test/query.js

```diff
@@ -1,568 +0,0 @@
-// *****************************************************************************
-// Copyright 2013-2022 Aerospike, Inc.
-//
-// Licensed under the Apache License, Version 2.0 (the "License")
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-// *****************************************************************************
-
-'use strict'
-
-/* eslint-env mocha */
-/* global expect */
-
-const Aerospike = require('../lib/aerospike')
-const Query = require('../lib/query')
-const Job = require('../lib/job')
-const helper = require('./test_helper')
-
-const AerospikeError = Aerospike.AerospikeError
-const GeoJSON = Aerospike.GeoJSON
-const Key = Aerospike.Key
-const filter = Aerospike.filter
-const op = Aerospike.operations
-
-const NUMERIC = Aerospike.indexDataType.NUMERIC
-const STRING = Aerospike.indexDataType.STRING
-const GEO2DSPHERE = Aerospike.indexDataType.GEO2DSPHERE
-const LIST = Aerospike.indexType.LIST
-const MAPVALUES = Aerospike.indexType.MAPVALUES
-const MAPKEYS = Aerospike.indexType.MAPKEYS
-
-const keygen = helper.keygen
-const metagen = helper.metagen
-const putgen = helper.putgen
-
-describe('Queries', function () {
```

Comment:
Any reason to remove all of these unit tests for queries, even though they don't use predexp or any other deprecated functionality?